src/components/work/overview/localizationslink/LocalizationsLink.module.css

```diff
@@ -0,0 +1,3 @@
+.linkstyle{
+    display: contents;
```

Comment:
Den var ny for mig :exploding_head:

Reply:
aner ikke hvad det betyder - men det virker fint